inorbit_republisher/scripts/republisher.py

```diff
@@ -109,6 +120,31 @@ def callback(msg, repub=repub):
         # subscribe
         subs[in_topic] = rospy.Subscriber(in_topic, msg_class, callback)
 
+    # Set-up static publishers
+    static_publishers = config.get('static_publishers', ())
+    for static_pub_config in static_publishers:
+        key = static_pub_config['out']['key']
+        topic = static_pub_config['out']['topic']
+
+        # If a literal value is provided, it takes highest precendence
+        val = static_pub_config.get('value')
+
+        # Otherwise, fetch the value from the specified source
+        if val is None:
+            value_from = static_pub_config.get('value_from')
+            if STATIC_VALUE_FROM_PACKAGE_VERSION in value_from:
+                pkg_name = value_from[STATIC_VALUE_FROM_PACKAGE_VERSION]
+                pkg_manifest = rospack.get_manifest(pkg_name)
```

Comment:
Shouldn't we add any exception handling here in case the manifest can't be found? Or at least leave a TODO.